### ISSUE TITLE
[Question] - Podcasts stored ouside soundwaves folder

### ISSUE THREAD

**Issue by smarquespt** [bottiger/SoundWaves#42]
Usually apps store its data in /internal/Android/Data/xxx but soundwaves stores its data in /internal/soundwaves and podcasts are downloaded to /internal/podcasts.

This should be intentional but whats the point of having /podcasts outside /soundwaves?

And why not follow what others do and store booth in /internal/Android/Data?

**Comment by bottiger** [comment]
The reason most of the data is on the "soundwaves" folder is just legacy, I do not remember why - and it should be fixed.

The reason for storing the podcasts in /internal/podcasts is to support Sonos. Sonos will only recognize a media files as a podcast if it's in the folder "podcasts".

I do not like it, but I also do not have a good solution for it.

**Comment by bottiger** [comment]
I have made a change which uses the /Android/Data/app.package for application data.
Reading the Android documentation I also learned that there is a [special directory for Podcasts](http://developer.android.com/reference/android/os/Environment.html#DIRECTORY_PODCASTS) which points to /internal/Podcasts, so i makes sense to store the downloaded files there.

However, this patch is rather huge, and touches many many different parts of the code. I will therefore not merge it into the master branch before I have used it myself for some time.
